apps/graphql/src/types/output/Itinerary.js

```diff
@@ -1,21 +1,56 @@
 // @flow
 
-import { GraphQLObjectType, GraphQLInt, GraphQLString } from 'graphql';
+import {
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLString,
+  GraphQLList,
+} from 'graphql';
 import GlobalID from '@kiwicom/graphql-global-id';
+import { GraphQLDateTime } from 'graphql-iso-date';
 
 export type Itinerary = {|
   +id: string,
 |};
 
+const tripSectorType = new GraphQLObjectType({
```

Comment:
What is our convention for GraphQL type definitions? I think that since `GraphQLInt` is in PascalCase, I would like our defined types to follow the same convention.